### PR TITLE
Extract attendee address logic out of the view

### PR DESCRIPTION
**Pull request by gjvis**
Cherry-picked this from the pdf-output repo, with a few minor changes to support the differences in this one.

### CODE REVIEW

File: app/templates/output_document_cover_letter.html.erb [hunk 1]
```diff
@@ -2,24 +2,7 @@
...
-      <%= output_document.attendee_address_line_3 %><br/>
-    <% end %>
-    <% if output_document.attendee_town %>
-      <%= output_document.attendee_town %><br/>
-    <% end %>
-    <% if output_document.attendee_county !='' %>
-      <%= output_document.attendee_county %><br/>
-    <% end %>
-    <%= output_document.attendee_postcode %>
+    <%= simple_format(output_document.attendee_address) %>
```
Comment (stevenwilkin) [inline app/templates/output_document_cover_letter.html.erb]: I didn't know about `simple_format`, very handy

File: app/templates/output_document_ineligible.html.erb [hunk 2]
```diff
@@ -2,24 +2,7 @@
...
-      <%= output_document.attendee_address_line_3 %><br/>
-    <% end %>
-    <% if output_document.attendee_town %>
-      <%= output_document.attendee_town %><br/>
-    <% end %>
-    <% if output_document.attendee_county !='' %>
-      <%= output_document.attendee_county %><br/>
-    <% end %>
-    <%= output_document.attendee_postcode %>
+    <%= output_document.attendee_address %>
```
Comment (stevenwilkin) [inline app/templates/output_document_ineligible.html.erb]: No use of `simple_format` here?

**Comment by stevenwilkin** [comment]
:+1:

File: app/templates/output_document_ineligible.html.erb (hunk 2 above)
Comment (gjvis) [inline app/templates/output_document_ineligible.html.erb]: Accidental.

**Pull request merged**